doc/releases/octopus.rst

```diff
@@ -5,6 +5,12 @@ v15.1.0 Octopus
 
 These are draft notes for the upcoming Octopus release.
 
+- *CephFS* distributed file system:
+
+  * The Linux kernel (as of release v5.4) supports a new
```

Comment:
This is my patch, but I'm not a fan of it.

First, this feature is not tied to any particular ceph release -- it should work just fine on any earlier release. Also, while we can note what upstream kernel release this feature went into, some distros (e.g. RHEL) are likely going to get a backport of it.

Personally, I think we ought to just limit this file to covering features that _actually_ went into octopus.

Reply:
I think we should evangelize new features in the kernel; the Ceph community may not keep an eye on the kernel release notes.

There's a CephFS section below this change should go into.

It's also good to add a note that the feature may be backported. Maybe we should just add a separate "kernel" section with a standard disclaimer.

Reply:
Oops, I misread the format of the file originally. It should be in the right place now.

As to whether and how to publish notes about the kclient, I don't know.

@idryomov: What do the kRBD folks do? Do they publish notes about that client in the userland release notes?

Reply:
No, generally we don't, perhaps with the exception of a significant change made on both sides.